service/controller/clusterapi/v29/ebs/ebs.go

```diff
@@ -146,7 +146,7 @@ func (e *EBS) ListVolumes(cr v1alpha1.Cluster, filterFuncs ...func(t *ec2.Tag) b
 	i := &ec2.DescribeVolumesInput{
 		Filters: []*ec2.Filter{
 			{
-				Name: aws.String(fmt.Sprintf("tag:%s", key.ClusterCloudProviderTag(cr))),
+				Name: aws.String(fmt.Sprintf("tag:%s", key.ClusterCloudProviderTag(&cr))),
```

Comment:
As Pawel mentioned in another PR we might go with CR pointers in the future when this is the new norm now.